README.md

```diff
@@ -128,6 +128,7 @@ A Nuxt module to automatically resize, rotate, blur, watermark and crop images.
 - [Guillermo Rauch - Static and Dynamic Next.js | JSHeroes 2017](https://youtu.be/lLNJsuXB4CI?t=36m50s)
 
 ### Showcase
+ - [homefinder.com](https://homefinder.com) - A nation-wide real estate portal in the US.  Find or or sell your home on HomeFinder!
```

Comment:
@limichange It should be at the end of the list, right?

Reply:
😗`Showcase` is ok. No problem.